exonum-java-binding/cryptocurrency-demo/pom.xml

```diff
@@ -205,6 +209,36 @@
           <skip>true</skip>
         </configuration>
       </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-failsafe-plugin</artifactId>
+        <configuration>
+          <includes>
+            <include>**/*IntegrationTest.java</include>
+          </includes>
+          <argLine>${jacoco.it.args}
+             -Djava.library.path=${ejb-core.nativeLibPath}
+            ${java.vm.assertionFlag}
+          </argLine>
+          <systemPropertyVariables>
+            <it.artifactFilename>${artifactFilename}.jar</it.artifactFilename>
+            <it.artifactId>${groupId}:${artifactId}:${version}</it.artifactId>
+            <it.serviceName>cryptocurrency-demo</it.serviceName>
+            <it.serviceId>46</it.serviceId>
+            <it.artifactsDirectory>target</it.artifactsDirectory>
```

Comment:
Please use the standard variable ${project.build.directory} so that it works regardless of the current directory.